content/Why you should use GatsbyJS to build your landing page - Nicolás Santos.md

```diff
@@ -0,0 +1,26 @@
+![Gatsby](images/gatsby.png)
+
+# Why you should use Gatsby to build your landing page
+Having an awesome landing page is essential for your business.It allows your potential clients to know about what problem do you resolve for them and it's the visible face of your company. For that you need your page loads as fast as possible, all the major search engines such as Google, Bing, and Yahoo have it primary in search results and make available for everyone. And **Gatsby takes care of that and much more for you.**
+
+## What is Gatsby?
+Gatsby is a free and open-source framework based on React that helps developers build blazing fast **websites** and  **apps**, or that is what they say on your website, for me Gatsby is more than other framework based on React, it's the way to build your website right from the start and solve all the classical problems made pages at the first line of code. Also, there is an amazing community behind them, contribute to make better and better Gatsby and create wonderful plugins that resolve all types of problems that enable integration with just a couple of code lines.
+
```

Comment:
- `that is what they say on _their_ website,`
- `_another_ framework based...`
- What do you mean here? `solve all the classical problems made pages at the first line of code.`
- `there is an amazing community behind them` `it` instead of `them`, since you are talking about the framework.
- `that contribute to make a better and better Gatsby`
- `lines of code`

Reply:
- I'd replace the comma for a period before saying your opinion about Gatsby.
- `In my opinion, Gatsby is much more than just another framework based on React...`
- `and create wonderful plugins that solve all types of problems with just a couple lines of code`. I don't know how I'd include the integration part.

Reply:
@missx I would change `solve all the classical problems made pages at the first line of code.` to `solve all the classical problems making pages with only a couple of lines of code`, WDYT?

Reply:
`solve all the classical problems by integrating the plugins pre-made by the community`?

Reply:
That's better, thanks!